tox.ini

```diff
@@ -15,14 +15,15 @@ python =
 [testenv]
```

Comment:
`[testenv:py{36,37,38,39,py3}-dj{master}]` needs to be excluded from the test matrix that is required to pass. At the moment installing `coverage` fails on Django master branch and that causes the whole test matrix to fail.

The following snippet should work for that:

```
[testenv:py{36,37,38,39,py3}-dj{master}]
ignore_errors = true
ignore_outcome = true
```

Reply:
Good catch!